modules/utxo-lib/src/noble_ecc.ts

```diff
@@ -68,7 +68,11 @@ const ecc = {
     throwToNull(() => necc.utils.pointAddScalar(p, tweak, defaultTrue(compressed))),
 
   privateAdd: (d: Uint8Array, tweak: Uint8Array): Uint8Array | null =>
-    throwToNull(() => necc.utils.privateAdd(d, tweak)),
+    throwToNull(() => {
```

Comment:
What's the purpose of this change?

Reply:
Fixes an error caused by `ECPair`'s (strongly against my preferences) `testecc` changes. Could use a comment.